jellyfin.ruleset

```diff
@@ -90,9 +135,16 @@
     <Rule Id="CA1303" Action="None" />
     <!-- disable warning CA1308: Normalize strings to uppercase -->
     <Rule Id="CA1308" Action="None" />
+    <!-- disable warning CA1848: Use the LoggerMessage delegates -->
+    <Rule Id="CA1848" Action="None" />
     <!-- disable warning CA2101: Specify marshaling for P/Invoke string arguments -->
     <Rule Id="CA2101" Action="None" />
     <!-- disable warning CA2234: Pass System.Uri objects instead of strings -->
     <Rule Id="CA2234" Action="None" />
   </Rules>
+
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.BannedApiAnalyzers" RuleNamespace="Microsoft.Design">
+    <!-- error on RS0030: Do not used banned APIs -->
+    <Rule Id="RS0030" Action="Error" />
```

Comment:
This rule is useless without the file with the banned APIs ;)
https://github.com/jellyfin/jellyfin/blob/master/BannedSymbols.txt
Also need to include it in the csproj file

Reply:
I included the package reference to `Microsoft.CodeAnalysis.BannedApiAnalyzers`, I didn't also need a file reference to `BannedSymbols.txt` did I?

Reply:
I think you do
https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.BannedApiAnalyzers/BannedApiAnalyzers.Help.md#how-to-use-microsoftcodeanalysisbannedapianalyzers

Reply:
Ah, I had done a search of the csproj files in jellyfin for references and didn't find any because it uses a different method to set defaults for all projects. Fixed it and tested that it correctly tests banned symbols this time.